ember_debug/libs/promise-assembler.js

```diff
@@ -6,8 +6,9 @@
  */
 
 import Promise from 'ember-debug/models/promise';
+import { copy } from 'ember-copy';
```

Comment:
@teddyzeenny should this import work here?

Reply:
I don't think it would, no. Not unless the app being debugged has `ember-copy` installed as an addon.

Reply:
So what is your recommendation here?

Reply:
There's no way to use Ember addons in `ember-debug` at the moment. We may be able to eventually work this out by custom-building and including the addons in the `ember-debug` final build, but I doubt it's going to be simple. 

For the sake of this PR, my recommendation would just be to not use `Ember.copy` in this file as it's not even doing a deep clone. So either `properties = {...properties}` if our Babel setup supports it, or a simple manual shallow copy by iterating over the properties. Either way, I don't think this function requires an Ember addon.